functions/exercise2/scriptSpec.js

```diff
@@ -1 +1,39 @@
 /// <reference path="../../typings/globals/jasmine/index.d.ts" />
+/*globals isPalindrome*/
+
+describe("Checking if word is palindrome: ", function () {
+    it("word tenet is a palindrome", function () {
+        var resultEntry = isPalindrome("tenet");
+        expect(resultEntry).toBeTruthy();
+    });
+    it("word dog is a palindrome", function () {
+        var resultEntry = isPalindrome("dog");
+        expect(resultEntry).toBeFalsy();
+    });
+     it("word kayak is a palindrome", function () {
+        var resultEntry = isPalindrome("kayak ");
+        expect(resultEntry).toBeTruthy();
+    });
+    it("word kayak is a palindrome", function () {
+        var resultEntry = isPalindrome(" kayak");
+        expect(resultEntry).toBeTruthy();
+    });
+    it("string kayak kayak is a palindrome", function () {
```

Comment:
Description is not correct - it should emphasise, that spaces inside the string provided are acceptable and crucial in determining whether it is a palindrome.